config/config-leader-election.yaml

```diff
@@ -0,0 +1 @@
+config/core/configmaps/leader-election.yaml
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
config/core/configmaps/leader-election.yaml

```

Reply:
That's how github presents symlinks, bot